samples/snippets/set_agent_test.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from google.api_core.exceptions import InvalidArgument
+
+import pytest
+
+from set_agent import set_agent
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+
+
+def test_set_agent():
+    with pytest.raises(InvalidArgument):
```

Comment:
Suggestion: The test code might confuse the readers. Maybe you can add a comment justifying the compromising test strategy.